cmd/snapd-apparmor/main.go

```diff
@@ -69,7 +69,7 @@ import (
 // be properly handled by this function.
 func isContainerWithInternalPolicy() bool {
 	if release.OnWSL {
-		return true
+		return false
```

Comment:
If this is just a workaround for now, perhaps a comment stating this should be changed once Microsoft fixes things would be good

Reply:
It's more of an "implements" things but you are right. I will push a comment.

Reply:
I've now included and pushed a comment.